Content.Shared/Doors/Components/DoorComponent.cs

```diff
@@ -258,6 +258,9 @@ private float? SecondsUntilStateChange
     [DataField, ViewVariables(VVAccess.ReadWrite)]
     public bool CanPry = true;
 
+    [DataField, AutoNetworkedField, ViewVariables(VVAccess.ReadOnly)]
+    public bool BeingPried;
```

Comment:
Minor nitpick: `IsBeingPried` reads a bit better.

Reply:
This was the original name of the property, you can even see this old name in the new CanChangeState() method, i changed it to BeingPried because i didn't see any other property using the Is{state} pattern, so i didn't want to change the convention. But i do prefer IsBeingPried as well. I'll change it to IsBeingPried then, thanks!